internal/pkg/runtime/c/lib/util/capability.c

```diff
@@ -21,3 +24,11 @@ int capget(cap_user_header_t hdrp, cap_user_data_t datap) {
 int capset(cap_user_header_t hdrp, const cap_user_data_t datap) {
     return syscall(__NR_capset, hdrp, datap);
 }
+
+#else
+
+int noop() {
```

Comment:
This source file is compiled regardless of platform or architecture, but we will get `AR`/`RANLIB` errors about not having any symbols so we suppress that by including a small no-op function.